src/components/Sheet/Mediasheet.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react'
+
+import { useMediaContext } from '@/context/MediaContext'
+import { Button } from '@/components/ui/button'
+import {
+  Sheet,
+  SheetContent,
+  SheetDescription,
+  SheetHeader,
+  SheetTitle,
+  SheetTrigger,
+} from '@/components/ui/sheet'
+import { Icons } from '../Icons'
+
+export const MediaSheet: React.FC = () => {
+  const { medias, currentMedia, setCurrentMedia, isPlaying, setIsPlaying, handleRemove } =
+    useMediaContext()
+
+  const displayText = (text: string) => {
+    if (text.length > 10) {
+      return text.slice(0, 2) + '...' + text.slice(-2)
+    } else {
+      return text
+    }
```

Comment:
else is redundant right?
```suggestion
    } 
    return text
```